.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: monthly
+# the rest handled by Renovate
```

Comment:
bit confused, renovate will also do github actions...

Reply:
Is it already configured to do so? If so, I can just delete the file. I am not familiar with what https://github.com/jenkinsci/reverse-proxy-auth-plugin/blob/be58e138d08eaf2b590f2266c01ff1012363ed22/.github/renovate.json#L4 would include.

Reply:
some links:
https://docs.renovatebot.com/presets-config/#configrecommended

https://docs.renovatebot.com/presets-group/#grouprecommended

its hard to tell from just looking there but AFAIK all languages are enabled by that, e.g. see Jenkins core:

https://github.com/jenkinsci/jenkins/pull/9963

https://github.com/jenkinsci/jenkins/blob/master/.github/renovate.json#L4